README.md

```diff
@@ -33,6 +33,11 @@ ws.on('pong', function incoming(data) {
 });
 ```
 
+## IP Rate limit
+- Receiving a 429 response will result in an automatic IP ban; it is your responsibility to refrain from spamming.
+- A `Retry-After` header is included in 429 responses, indicating the Unix timestamp in seconds for when you can reconnect.
+- Rate limit violations will result in a penalty. You must wait for the full period of the penalty before making any subsequent requests. Failure to yield to the penalty will result in a reset of the penalty period.
```

Comment:
How about making it more explicit?

```
- Rate limit violations will result in a penalty. You must wait for the full period of the penalty before making any subsequent requests. Otherwise, the penalty time will be reset and the period of being banned will be prolonged.
```

Reply:
nice!